code/modules/paperwork/faxmachine.dm

```diff
@@ -237,25 +254,20 @@ GLOBAL_LIST_EMPTY(fax_blacklist)
 		return
 
 	if(scan)
-		to_chat(usr, "You remove \the [scan] from \the [src].")
+		to_chat(usr, "You remove [scan] from [src].")
 		scan.forceMove(get_turf(src))
 		if(!usr.get_active_hand() && Adjacent(usr))
 			usr.put_in_hands(scan)
 		scan = null
 	else
-		to_chat(usr, "There is nothing to remove from \the [src].")
+		to_chat(usr, "There is nothing to remove from [src].")
 
 /obj/machinery/photocopier/faxmachine/proc/sendfax(var/destination,var/mob/sender)
-	if(stat & (BROKEN|NOPOWER))
```

Comment:
I assume this was removed because you checked it at the `tgui_act` level?

Reply:
Yes